src/audio/copier/copier_dai.c

```diff
@@ -259,7 +259,7 @@ int copier_dai_create(struct comp_dev *dev, struct copier_data *cd,
 		break;
 	case ipc4_i2s_link_output_class:
 	case ipc4_i2s_link_input_class:
-		dai_index[dai_count - 1] = (dai_index[dai_count - 1] >> 4) & 0xF;
+		dai_index[dai_count - 1] = node_id.f.v_index;
```

Comment:
line 247 I can see this:

dai_index[dai_count - 1] = node_id.f.v_index;

So this new assignment is redundant?

Reply:
correct, this is redundant. I will remove it in the next update (This PR was created mainly to check the CI results - I missed this). Thanks